frontend-app/src/amit-ui-task/index.html

```diff
@@ -0,0 +1,104 @@
+                            <a href="https://medium.com/@theinterned">Ned Schwartz</a> 
+                            <button class= "follow pointer">Follow</button>
+                            <img id="mail" class="pointer" src="./images/Screenshot 2022-02-06 at 11.33.49 PM.png" alt="" style="height: 30px; width: 30px;">
+                            
+                        </div>
+                        <div class="text">
+                            <a href="">Feb 6, 2017</a>
+                            <span>· 6 min read</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="social">
+                    <a href=""><img src="./images/twt.png"/></a> 
+                    <a href=""><img src="./images/fb.png"/></a>
+                    <a href=""><img src="./images/linkedin.png" alt=""></a>
+                    <a href=""><img src="./images/link.png" alt=""></a>
+                    <a href=""><img src="./images/bookmark.png" alt=""></a>
+                    <a href=""><img src="./images/more.png" alt=""></a>
+                </div>
+            </div>
+            <div style="display: flex; justify-content: space-around; margin-top: 20px;">
+                <img src="./images/img.png" alt="">
+            </div>
+            <div >
+                <p style="font-size: 21px;"> <i> <b> Update January 18th, 2018:</b> CSS Module support has landed in alpha builds of Create React App v2 🎉. </i> </p>
+            </div>
+        </div>
+
+
+    <div class="modal" id = "login-modal">
+        <div class="modal-content">
+            <span class="close pointer" onclick= "closemodal()" >&times;</span>
+            <h1 class="modal-heading" >Welcome Back</h1>
+            
+            <button class="sign-in-opt pointer"> <img class="icon" src="https://img.icons8.com/color/20/000000/google-logo.png"/> Sign in with Google</button>
+            <button class="sign-in-opt pointer"><img class="icon" src="https://img.icons8.com/fluency/20/000000/facebook.png"/>Sign in with Facebook</button>
+            <button class="sign-in-opt pointer"><img class="icon" src="https://img.icons8.com/ios-glyphs/20/000000/mac-os.png"/>Sign in with Apple</button>
+            <button class="sign-in-opt pointer"><img class="icon" src="https://img.icons8.com/color/20/000000/twitter--v1.png"/>Sign in with Twitter</button>
+            <button class="sign-in-opt pointer"><img class="icon" src="https://img.icons8.com/external-kiranshastry-lineal-kiranshastry/20/000000/external-email-advertising-kiranshastry-lineal-kiranshastry-1.png"/>Sign in with Email</button>
+
+            <p class="signup">No account? <a href=""> Create one </a> </p>
+            <p class="description">Click “Sign In” to agree to Medium’s <a href=""> Terms of Service </a> and acknowledge that Medium’s <a href=""> Privacy Policy </a>applies to you.</p>
+        </div>
+    </div>
+    <div class="modal" id = "signup-modal">
```

Comment:
- signup modal positioning should in centre
- cross sign should be at top right corner of the modal
- cross sign thickness should be less